onnxruntime/test/platform/ios/ios_package_test/README.md

```diff
@@ -0,0 +1,28 @@
+# iOS End-to-End Test App for ORT-Mobile
+
+This End-to-End test app for iOS will test ORT Mobile C/C++ API framework using CoocaPods
+
+## Requirements
+
+- [CocoaPods](https://cocoapods.org/)
+
+## Build and Test iOS Framework using build.py
+
```

Comment:
Is there a recommended/required version of XCode or other tools that should be used?